### DISCUSSION TITLE
question how refresh works

### DISCUSSION THREAD

**Discussion by kangaroolab** [vanjs-org/van#380] [General]
if I have a code like below where I have static content and dynamic content, doe vanjs refresh them every time together or only refresh the dynamic party?

```
return div(
  div("static content"),
  div(variable.val)
)
```
if I like to not refresh static content, do I have to separate it out?

**Comment by sirenkovladd** [comment]
Your code refresh together

you can change it to
```js
return div(
  div("static content"),
  div(variable)
// or
  () => div(variable.val)
// or
  div(() => variable.val)
)
```

and the static part will not be refresh

**Comment by kangaroolab** [comment]
thanks...

is div(variable.val) same as div(variable)?

**Reply by sirenkovladd** [reply]
not exactly

`div(variable.val)` this more similar to `div("static content")`

`div(variable)` to `div(() => variable.val)`

when you pass a van.state as an argument, van automatically makes a binding, preventing the parent element from being re-rendered
